ci/travis/install-dependencies.sh

```diff
@@ -216,8 +216,10 @@ install_upgrade_pip() {
 }
 
 install_node() {
-  if [ "${OSTYPE}" = msys ]; then
+  if [ "${OSTYPE}" = msys ] ; then
     { echo "WARNING: Skipping running Node.js due to incompatibilities with Windows"; } 2> /dev/null
+  elif [ -n "${BUILDKITE-}" ] ; then
+    { echo "WARNING: Skipping running Node.js on buildkite because it's already there"; } 2> /dev/null
```

Comment:
Where is this happening?

Reply:
few lines above in install nvm. we end up using apt-get for buildkite